src/Password/PasswordService.php

```diff
@@ -3,15 +3,14 @@
 namespace Corcel\Password;
 
 use Hautelook\Phpass\PasswordHash;
-use Illuminate\Hashing\BcryptHasher;
 
 require_once(__DIR__ . "/../../vendor/autoload.php");
 
 class PasswordService
 {
     function __construct()
     {
-        $this->wp_hasher = new BcryptHasher();
+        $this->hasher = new PasswordHash(8, false);
```

Comment:
[2nd param should be true](https://github.com/WordPress/WordPress/blob/a311bea6d11bceabfff96ed03a9908ff7ba10505/wp-includes/pluggable.php#L2005-2015), to get the same behaviour as wp